Chat_Log/Commands/LocalRegister.cs

```diff
@@ -14,9 +14,9 @@ public LocalRegister()
             CommandHandler.RegisterCommand("MiniLog", new Action<ShPlayer, string>(LocalCommand));
         }
 
-        private void LocalCommand(ShPlayer player, string Type)
+        private void LocalCommand(ShPlayer player, string type)
         {

```

Comment:
I think "type" without caps in the parameters will make things better when reading the code.